cw-orch-daemon/src/sender.rs

```diff
@@ -61,6 +61,8 @@ pub struct Sender<C: Signing + Context> {
 pub struct SenderOptions {
     pub authz_granter: Option<String>,
     pub fee_granter: Option<String>,
+    /// Wallet mnemonic
+    pub mnemonic: Option<String>,
```

Comment:
Do we need this here? A bit confusing to have both a `Sender` and a mnemonic

Reply:
Yes. It's done to satisfy 2 use cases : 
- Building a daemon with a custom mnemonic
- Allowing a daemon to be `rebuilt` from an existing sender

One solution to avoid this could be to construct the sender as soon as the mnemonic is given but that goes against the builder principle.
Another one would be to be able to extract all senderOptions from the built sender (which is not possible today because the mnemonic for instance is not stored)

Reply:
How about this, we move `mnemonic` out of `SenderOptions` and create an enum 

```rust
enum Wallet {
  Sender(Sender),
  Mnemonic(String),
}
```

And have a field in the builder with this

```rust
struct Builder {
  wallet: Option<Wallet>
}
```

When re-building this wallet is cloned (and it will be Sender type) 

If you want to set a new mnemonic you use the `.mnemonic` on the builder to overwrite the sender object.